src/components/LoansByCategory/HelpmeChoose/HelpmeChooseRecommendations.vue

```diff
@@ -115,6 +124,25 @@ export default {
 	methods: {
 		selectLoan(evt) {
 			this.selectedLoan = evt;
+			let eventProperty = '';
+			switch (evt) {
+				case 0:
+					eventProperty = 'first';
+					break;
+				case 1:
+					eventProperty = 'second';
+					break;
+				case 2:
+					eventProperty = 'third';
+					break;
+				default:
+					eventProperty = '';
+			}
+			this.$kvTrackEvent(
```

Comment:
This event is way too generic. 'event-tracking', 'click', 'first' does not describe what is happening. 

Maybe 'loan-card', 'show', 'help-me-choose-loan-1'